test/unit/app/renderer/downloadItemTest.js

```diff
@@ -53,13 +50,19 @@ describe('downloadItem component', function () {
       before(function () {
         this.downloadId = uuid.v4()
         this.download = newDownload(state)
-        this.result = mount(<DownloadItem downloadId={this.downloadId} download={this.download} downloadDeleteConfirmationVisible={this.download.downloadDeleteConfirmationVisible} />)
+        this.result = mount(
+          <DownloadItem
+            downloadId={this.downloadId}
+            download={this.download}
+            deleteConfirmationVisible={this.download.get('deleteConfirmationVisible')} />
```

Comment:
NOTE: this was wrong before.  Before, it was `this.download.downloadDeleteConfirmationVisible`. Accessing child objects from an Immutable object will return `undefined`.

I updated this to use the Immutable.js `get` method to fetch the variable